Products/SQLAlchemyDA/da.py

```diff
@@ -451,10 +452,10 @@ def manage_doQuery(self, query):
     security.declareProtected(view_management_screens, 'manage_formatItem')
     def manage_formatItem(self, s):
         """ used by query form """
-        if isinstance(s, unicode):
-            return s
-        if isinstance(s, str):
-            return unicode(s, self.encoding, 'ignore')
+        # if isinstance(s, unicode):
+        #    return s
+        # if isinstance(s, str):
+        #    return unicode(s, self.encoding, 'ignore')
```

Comment:
The intent here was to leave unicode strings untouched and convert binary strings to unicode in a safe way. Removing all that invites `UnicodeDecodeError` on the subsequent `str` call.